packages/components/src/QueryModel/QueryModel.ts

```diff
@@ -277,6 +277,19 @@ export class QueryModel {
         });
     }
 
+    /**
+     * Returns the data for the specified key parameter on the QueryModel.rows object.
+     * If no key parameter is provided, the first data row will be returned.
+     * @param key
+     */
+    getRow(key?: string): any {
+        if (key === undefined && this.rows) {
+            key = Object.keys(this.rows)[0];
+        }
+
+        return this.rows[key];
```

Comment:
If `this.rows` is undefined, bad things would happen here.

Reply:
Good call. Fixed.